rcgen/src/key_pair.rs

```diff
@@ -206,14 +217,15 @@ impl KeyPair {
 		Ok(KeyPair {
 			kind,
 			alg,
-			serialized_der: pkcs8_vec,
+			serialized_der,
 		})
 	}
 
 	#[cfg(feature = "crypto")]
 	pub(crate) fn from_raw(
-		pkcs8: &[u8],
+		pkcs8: &PrivatePkcs8KeyDer,
```

Comment:
Could you also mention in the docs that one can pass a byte slice and use `into` maybe with an example?